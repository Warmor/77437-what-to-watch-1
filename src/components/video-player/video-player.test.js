import React from 'react';
import Enzyme, {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import {shallowToJson} from 'enzyme-to-json';
Enzyme.configure({adapter: new Adapter()});

import VideoPlayer from "~/components/video-player/video-player.jsx";

describe(`<VideoPlayer/>`, () => {
  it(`WelcomeScreen renders correctly`, () => {
    const mock = {
      src: `src:path`,
      poster: `poster:path`,
    };
    const tree = shallow(<VideoPlayer src={mock.src} poster={mock.poster} muted controls/>);
    expect(shallowToJson(tree)).toMatchSnapshot();
  });
});
