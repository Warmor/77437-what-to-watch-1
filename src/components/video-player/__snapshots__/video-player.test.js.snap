// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VideoPlayer/> WelcomeScreen renders correctly 1`] = `
<video
  autoPlay={true}
  controls={true}
  height="175"
  muted={true}
  poster="poster:path"
  src="src:path"
  width="280"
/>
`;
